test/localrunner.html

```diff
@@ -3,7 +3,7 @@
     <meta charset="utf-8">
     <!-- encoding must be set for mocha's special characters to render properly -->
     <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
-    <script src="./vendor/lodash.min.js"></script>
+    <script src="../lodash.custom.min.js"></script>
```

Comment:
Is there a way we can package lodash w/ the webpacked ripple-lib, so that clients don't have to import our custom lodash package?

Reply:
Pretty sure that already happens with the custom file we have - when it's packaged it gets packaged based on our custom settings.

Reply:
Iirc we expect clients to import lodash separately: https://xrpl.org/get-started-using-node-js.html#rippleapi-in-web-browsers.

If we wanna package lodash w/ ripple-lib, we can do that, but I think wed have to adjust the webpacking.

Reply:
I looked into that - the file is like 70 kb bigger if we package lodash with it :(

I think it's fine for now and we can revisit the issue later

Reply:
Sounds good. We should definitely revisit as a part of #1222